bindings/go/credentials/spec/config/runtime/merge.go

```diff
@@ -3,7 +3,7 @@ package runtime
 // Merge merges the provided configs into a single config.
 func Merge(configs ...*Config) *Config {
 	if len(configs) == 0 {
-		return nil
+		return &Config{}
```

Comment:
Just saying that this means that we will have to check in a different way that things aren't set.

Reply:
>what are you referring to?

That now you can't do `== nil` you have to figure out what the zero value is for the struct.